RInAppMessaging/Classes/Router.swift

```diff
@@ -75,10 +89,57 @@ internal class Router: RouterType {
                     completion(true)
                     return
                 }
-                view.show(accessibilityCompatible: self.accessibilityCompatibleDisplay, onDismiss: {
+                guard let rootView = self.getKeyWindow(),
+                      self.findPresentedIAMView(from: rootView) == nil else {
+                    return
+                }
+
+                var parentView: UIView = rootView
+
+                // For accessibilityCompatible option, campaign view must be inserted to
+                // UIWindow's main subview. Private instance of UITransitionView
+                // shouldn't be used for that - that's why it's omitted.
+                if self.accessibilityCompatibleDisplay,
+                    let mainSubview = rootView.subviews.first(
+                        where: { !$0.isKind(of: NSClassFromString("UITransitionView")!) }) {
+                    parentView = mainSubview
+                }
+
+                view.show(accessibilityCompatible: self.accessibilityCompatibleDisplay,
+                          parentView: parentView,
+                          onDismiss: {
                     completion(false)
                 })
             }
         }
     }
+
```

Comment:
Could the below helper funcs be refactored into unit-testable UIView/UIWindow extensions?

Reply:
Done.